src/components/dashboard/constants.js

```diff
@@ -46,4 +46,7 @@ export const XS_NUM_COLUMNS = 1;
 export const XS_PIXELS = 0;
 
 export const USER_SURVEY = "https://cyver.se/srvy";
-export const PHX_BIO_URL = "https://www.phoenixbioinformatics.org/";
\ No newline at end of file
+export const PHX_BIO_URL = "https://www.phoenixbioinformatics.org/";
+export const FEATURE_MATRIX_URL = "https://cyverse.org/subscribe";
```

Comment:
This page doesn't exist yet but it will be available before this change goes to prod.